src/CallInvite.tsx

```diff
@@ -55,46 +72,149 @@ export declare interface CallInvite {
    */
 
   /**
-   * MessageReceived event. Raised when {@link (CallMessage:class)} is received.
+   * Accepted event. Raised when the call invite has been accepted.
+   *
    * @example
-   * ```typescript
-   * callInvite.addListener(CallInvite.Event.MessageReceived, (message) => {
-   *    // callMessage received
-   * })
+   * ```ts
+   * voice.on(Voice.Event.CallInvite, (callInvite) => {
+   *   callInvite.on(CallInvite.Event.Accepted, () => {
+   *     // the call invite was accepted through either the native layer
+   *     // or the js layer
+   *   });
+   * });
    * ```
    *
-   * @param messageReceivedEvent - The raised event string.
+   * @remarks
+   *
+   * @param acceptedEvent - The raised event string.
    * @param listener - A listener function that will be invoked when the event
    * is raised.
-   * @returns - The callMessage object
+   * @returns - The call invite object.
    */
   addListener(
-    messageReceivedEvent: CallInvite.Event.MessageReceived,
-    listener: CallInvite.Listener.MessageReceived
+    acceptedEvent: CallInvite.Event.Accepted,
+    listener: CallInvite.Listener.Accepted
   ): this;
   /** {@inheritDoc (CallInvite:interface).(addListener:1)} */
   on(
-    callMessageEvent: CallInvite.Event.MessageReceived,
-    listener: CallInvite.Listener.MessageReceived
+    acceptedEvent: CallInvite.Event.Accepted,
+    listener: CallInvite.Listener.Accepted
+  ): this;
+
+  /**
+   * Rejected event. Raised when the call invite has been rejected.
+   *
+   * @example
+   * ```ts
+   * voice.on(Voice.Event.CallInvite, (callInvite) => {
+   *   callInvite.on(CallInvite.Event.Rejected, () => {
+   *     // the call invite was rejected through either the native layer
+   *     // or the js layer
+   *   });
+   * });
+   * ```
+   *
+   * @remarks
+   *
+   * @param rejectedEvent - The raised event string.
+   * @param listener - A listener function that will be invoked when the event
+   * is raised.
+   * @returns - The call invite object.
+   */
+  addListener(
+    rejectedEvent: CallInvite.Event.Rejected,
+    listener: CallInvite.Listener.Rejected
+  ): this;
+  /** {@inheritDoc (CallInvite:interface).(addListener:2)} */
+  on(
+    rejectedEvent: CallInvite.Event.Rejected,
+    listener: CallInvite.Listener.Rejected
+  ): this;
+
+  /**
+   * Cancelled event. Raised when the call invite has been cancelled.
+   *
+   * @example
+   * ```ts
+   * voice.on(Voice.Event.CallInvite, (callInvite) => {
+   *   callInvite.on(CallInvite.Event.Cancelled, () => {
```

Comment:
Do you want to add error here as argument?
```suggestion
   *   callInvite.on(CallInvite.Event.Cancelled, (error) => {
```